lte/gateway/c/core/oai/CMakeLists.txt

```diff
@@ -129,6 +129,9 @@ include_directories(${PROJECT_SOURCE_DIR}/include)
 # Add magma libraries
 #############################################################
 
+set(MAGMA_COMMON_INCL_DIR $ENV{C_BUILD}/common)
```

Comment:
This is incorrect you should expose this as a target and import the target through the find_package(MAGMA_SENTRY) command, it should bring in the include directories itself. Also note include_directories is kinda retired in CMake the pratice is to use target_include_directories

Reply:
Done. `find_package` was key. Much simpler cmake changes. Thanks!